content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -75,7 +75,35 @@ In [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) and later, you
 
 <!-- todo this can be formatted nicer, and not sure this is the correct TS client syntax -->
 
-Additionally, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API. Specifying `@@id([firstName, lastName], name: "fullName")` will allow you to do `prisma.user.findUnique({where: {fullName:{firstName: "Paul", lastName:"Panther"}}})` instead of `prisma.user.findUnique({where: {firstName_lastName:{firstName: "Paul", lastName:"Panther"}}})`.
+Additionally, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API. Specifying
+`@@id([firstName, lastName], name: "fullName")` will change the API the client exposed to uniquely identify entities.
+
+```prisma
+prisma
+  .user
+  .findUnique({
+    where: {
+      fullName:{
+        firstName: "Paul",
+        lastName:"Panther"
+      }
+    }
+  })
+
+instead of
```

Comment:
Could you make this a comment, otherwise it looks like its supposed to be code

Reply:
We could also make this a diff... the `firstName_lastName` is replaced by `fullName`.

(There should als obe a space after the `:` everywhere in the code example)

Can you maybe type that up and add it as a suggestion @molebox? You know the docs syntax of a diff best I think.

Reply:
ok

Reply:
Something like this maybe? 

````
<TabbedContent tabs={[<FileWithIcon text="Using @@id and the name argument" icon="code"/>, <FileWithIcon text="Without the name argument" icon="code"/>]}> 
 
<tab> 

 ```ts
prisma
  .user
  .findUnique(
    where: {
      fullName:{
        firstName: "Bruce",
        lastName: "Banner"
      }
    }
  })
```
 
</tab> 
 
<tab> 

```ts 
prisma
  .user
  .findUnique({
    where: {
      firstName_lastName:{
        firstName: "Bruce",
        lastName: "Banner"
      }
    }
  })
``` 

</tab> 

</TabbedContent>
````

Reply:
Hm I am not sure if tabs are the best solution. I really meant 1 code block with the both lines, before and after, and then highlights them red and green as before and after.

Reply:
Oh wow. Yep, that would be a much cleaner solution!

Reply:
https://github.com/prisma/docs/pull/2159/files#discussion_r692788964